packages/react/src/components/Video/Video.tsx

```diff
@@ -0,0 +1,106 @@
+import * as PropTypes from 'prop-types'
+import * as React from 'react'
+
+import { createShorthandFactory, UIComponent, UIComponentProps, commonPropTypes } from '../../lib'
+
+import { ReactProps } from '../../types'
+import { defaultBehavior } from '../../lib/accessibility'
+import { Accessibility } from '../../lib/accessibility/types'
+import Ref from '../Ref/Ref'
+
+export interface VideoProps extends UIComponentProps {
+  /** Whether the video should start playing when rendered. Autoplay videos must be muted or they will not play immediately in certain browers like Chrome. */
+  autoPlay?: boolean
+
+  /** Whether to display the native video controls. */
+  controls?: boolean
+
+  /** Whether the video should automatically restart after it ends. */
+  loop?: boolean
+
+  /** Whether the video should be allowed to play audio. */
+  muted?: boolean
+
+  /** Image source URL for when video isn't playing. */
+  poster?: string
+
+  /** Video source URL. */
+  src?: string
+}
+
+/**
+ * An video is a graphicical and audio representation of something.
+ */
+class Video extends UIComponent<ReactProps<VideoProps>> {
+  static create: Function
+
+  static className = 'ui-video'
+
+  static displayName = 'Video'
+
+  static propTypes = {
+    ...commonPropTypes.createCommon({
+      children: false,
+      content: false,
+    }),
+    autoPlay: PropTypes.bool,
+    controls: PropTypes.bool,
+    loop: PropTypes.bool,
+    muted: PropTypes.bool,
+    poster: PropTypes.string,
+    src: PropTypes.string,
+  }
+
+  static defaultProps = {
+    as: 'video',
+    accessibility: defaultBehavior as Accessibility,
```

Comment:
No need for this, as the `defaultBehavior` will be applied if nothing is provided here.

Reply:
We can currently an assertion for this, going to address it in a separate PR.